### PR TITLE
i18n(pt-PT): update outdated pages

### PR DESCRIPTION
**Pull request by rgilsimoes**
#### Description

- Updated getting-started.mdx community-content.mdx and plugins.mdx

### CODE REVIEW

File: docs/src/content/docs/pt-pt/resources/community-content.mdx [hunk 1]
```diff
@@ -125,5 +134,19 @@ import YouTubeGrid from '~/components/youtube-grid.astro';
 			description:
 				'Tenha um novo site Starlight a funcionar em cerca de 5 minutos',
 		},
+		{
+			href: 'https://www.youtube.com/watch?v=12o7WxjAxjM',
+			title:
+				'Inclua a documentação Startligth num projecto Next.js através de proxies',
```
Comment (delucis) [inline docs/src/content/docs/pt-pt/resources/community-content.mdx:140]: Typo here I think
```suggestion
				'Inclua a documentação Starlight num projecto Next.js através de proxies',
```

File: docs/src/content/docs/pt-pt/resources/community-content.mdx [hunk 2]
```diff
@@ -125,5 +134,19 @@ import YouTubeGrid from '~/components/youtube-grid.astro';
...
+			description:
+				'Configurar o Startlight como uma sub-directoria dentro de um site Next.js',
```
Comment (delucis) [inline docs/src/content/docs/pt-pt/resources/community-content.mdx:142]: And here
```suggestion
				'Configurar o Starlight como uma sub-directoria dentro de um site Next.js',
```

File: docs/src/content/docs/pt-pt/resources/community-content.mdx [hunk 3]
```diff
@@ -125,5 +134,19 @@ import YouTubeGrid from '~/components/youtube-grid.astro';
...
+		},
+		{
+			href: 'https://www.youtube.com/watch?v=5pq80drDrNs',
+			title:
+				"Eu recreei a documentação do Shadcn's em 30 minutoes com esta ferramenta fantástica",
+			description:
+				'Neste video eu abordo o que torna o Starligh tão porreiro e porque razão deve experimentá-lo no seu próximo projecto.',
```
Comment (delucis) [inline docs/src/content/docs/pt-pt/resources/community-content.mdx:149]: And also here

```suggestion
				'Neste video eu abordo o que torna o Starlight tão porreiro e porque razão deve experimentá-lo no seu próximo projecto.',
```

**Review by delucis — CHANGES_REQUESTED** [review]
Thank you for the update @rgilsimoes! Just spotted a couple of small typos.

**Comment by rgilsimoes** [comment]
> Thank you for the update @rgilsimoes! Just spotted a couple of small typos.

Darn... dislexia hit me hard on Starlight :-)

**Review by delucis — APPROVED** [review]
English spelling sure doesn't help! 

Thank you @rgilsimoes 💖

**Pull request merged**